pkg/kubelet/network/plugins.go

```diff
@@ -120,3 +135,7 @@ func (plugin *noopNetworkPlugin) SetUpPod(namespace string, name string, id kube
 func (plugin *noopNetworkPlugin) TearDownPod(namespace string, name string, id kubeletTypes.DockerID) error {
 	return nil
 }
+
+func (plugin *noopNetworkPlugin) Status(namespace string, name string, id kubeletTypes.DockerID) (*NetworkStatus, error) {
+	return nil, nil
```

Comment:
This does not seem to handled by kubelet - I expect a nil dereference.  It should either handle a nil return and fall back on docker inspect or this should be a required (and tested) non-nil return.